components/webext-storage/README.md

```diff
@@ -0,0 +1,94 @@
+# WebExtension Storage Component
+
+The WebExtension Storage component can be used to power an implementation of the
+[`chrome.storage.sync`](https://developer.chrome.com/extensions/storage) WebExtension API,
+which gives each WebExtensions its own private key-value store that will sync between a user's
+devices. This particular implementation sits atop [Firefox Sync](../sync_manager/README.md).
+
+* [Features](#features)
+* [Using the Logins component](#using-the-component)
+* [Working on the Logins component](#working-on-the-component)
+
+## Features
+
+The WebExtension Storage component offers:
+
+1. Local storage of key-value data indexed by WebExtension ID.
+1. Basic Create, Read, Update and Delete (CRUD) operations for items in the database.
+1. Syncing of stored data between applications, via Firefox Sync.
+1. Import functionality from previous  WebExtension storage implementations backed by
```

Comment:
Importing isn't currently done by this component. It might be, but the current WIP just has JS code in desktop reading the old DB and using the existing public API to write it.